README.md

```
@@ -4,6 +4,77 @@
 
 ## EatnGo 용어 정리
 
+# EatnGo
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content
null

(MD024, no-duplicate-heading)